README.md

```diff
@@ -0,0 +1,30 @@
+> NOTE: We have recently made significant changes to our repository structure. In order to 
+streamline our development process and foster better contributions, we have merged three separate 
+repositories Cumulus, Substrate and Polkadot into this repository. Read more about the changes [
+here](https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f).
+
+# Polkadot SDK
+
```

Comment:
Where would you want to place these links? Just at the bottom of the readme, or already in the intro text?

Reply:
So the runtimes repo https://github.com/polkadot-fellows/runtimes/ will hold the Polkadot and Kusama runtime, that is quite important.  
For the RFC it is more R&D like, maybe mentioning that this can be used to see upcoming ideas https://github.com/polkadot-fellows/RFCs/  
And the XCM i guess leave it out for now and wait until it is stabilized.

Reply:
@ggwpez I've added a section with links to both repositories. If you have the time please review them for correctness :)